src/pages/learn/[id].tsx

```diff
@@ -74,7 +73,33 @@ function LearnDetail({ highlightData }: { highlightData: HighlightData[] }) {
     return { x: x + x * 0.5, y: y + y * 0.5 };
   };
 
+  //하이라이트 인덱스 구하는 함수입니다.
+  //클릭된 하이라이트의 부모 노드와 해당 하이라이트의 텍스트를 인자로 받습니다.
+  //부모노드를 순회하면서 하이라이트 텍스트와 같은 내용이 나올때까지 글자수를 카운트하면서 증가시킵니다.
+  const [highlighIdx, setHighlightIndex] = useState<number>(0);
```

Comment:
```suggestion
  const [highlightIndex, setHighlightIndex] = useState<number>(0);
```

살짝 오타가 있네용 👀
그리고 앞에서는 idx, 뒤에서는 index 이렇게 쓰지 말고 **index** 하나로 통일 부탁드립니다~!